content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -344,20 +341,20 @@ overall maximum that you configured).
 
 ## Support for metrics APIs
 
-By default, the HorizontalPodAutoscaler controller retrieves metrics from a series of APIs.  In order for it to access these
+By default, the HorizontalPodAutoscaler controller retrieves metrics from a series of APIs. In order for it to access these
 APIs, cluster administrators must ensure that:
 
-* The [API aggregation layer](/docs/tasks/extend-kubernetes/configure-aggregation-layer/) is enabled.
+- The [API aggregation layer](/docs/tasks/extend-kubernetes/configure-aggregation-layer/) is enabled.
 
-* The corresponding APIs are registered:
+- The corresponding APIs are registered:
```

Comment:
I'm not sure why we prefer '-' over '*'?

Reply:
It sees that there is no differences between dashes(-) and asterisks(*).

Refer to https://www.markdownguide.org/basic-syntax/#unordered-lists.

Should I revert them?

Reply:
There is no need to revert, but please avoid this kind of tweaks next time.

Reply:
Thanks, keep in my mind.